mslib/msui/linearview.py

```diff
@@ -91,16 +96,26 @@ def __init__(self, parent=None, model=None, _id=None):
 
         # Dock windows [WMS]:
         self.cbTools.clear()
-        self.cbTools.addItems(["(select to open control)", "Linear Section WMS"])
-        self.docks = [None]
+        self.cbTools.addItems(["(select to open control)", "Linear Section WMS", "MSS AUTOPLOT"])
```

Comment:
all components are for MSS, maybe name it just Autoplot, the other also not have Uppercase.